### DISCUSSION TITLE
Is this library still in development?

### DISCUSSION THREAD

**Discussion by RollsChris** [mojs/mojs#268] [Q&A] [answered]
Hey 

just wondering why i can still see important pages missing on the website. An example is:
https://mojs.github.io/tutorials/easing/path-easing/

I want to get serious and learn an animation library, wondering if this is a good choice.

Thanks

**Comment by xavierfoucrier** [comment, marked as answer]
Hi @RedEyedDog,

Thanks for posting your question here.
The MoJS project is not currently in "active" / "intensive" development, but it is not archived.

We - @xavierfoucrier & @Sandstedt - have inherited this project from the author developer years ago.
That's why there could be some missing part in the documentation.

Since 2019, the ecosystem is still **maintained and developed** in order to keep dependencies up to date and have as less security vulnerabilities as possible, which is really important for us. We work on the project on our free time, have any sponsors since the beginning, are working together in private companies and finally are working on other open source projects too. That's why it's sometimes hard to focus and make a project move forward.

Anyway, if you want to start learning an animation library, **MoJS is a good starting point**. You will find anything you need in the current documentation for basic / advanced animations. I also suggest you to have a look at other animations library, because most looks the same, but may have different internal behavior / features.

Here is a non-exhaustive list:
- gsap — https://greensock.com/gsap/
- popmotion — https://popmotion.io/
- animejs — https://animejs.com/
- spirit — https://spiritjs.io/
- theatrerjs — https://www.theatrejs.com/

Thanks for your interest in this project! :tada: 

> Feel free to have a look at sponsoring here:
> https://github.com/sponsors/xavierfoucrier
